packages/sdk/src/sdk/services/Solana/programs/types.ts

```diff
@@ -27,15 +27,48 @@ export const PrioritySchema = z.enum([
   'UNSAFE_MAX'
 ])
 
+// Note: Don't just use a custom object w/ `instanceof TransactionInstruction`
+// as mismatching versions of @solana/web3.js would fail the instanceof check.
```

Comment:
can we make sure to make all solana/web3.js versions the same across the stack?

Reply:
yeah I double checked, the only places it differs is in wormhole sdk deps that are a bit ahead (but those are dev deps anyway). This comment is a bit misleading, it's more about different _instances_ of @solana/web3.js not versions (SDK bundles @solana/web3.js I think so its using a separate one iiuc?)